apps/web-giddh/src/app/new-inventory/component/adjust-inventory/adjust-inventory.component.html

```diff
@@ -0,0 +1,407 @@
+                                            {{
+                                                adjustInventoryCreateEditForm.get("adjustmentMethod")?.value
+                                                    ? adjustInventoryCreateEditForm.get("adjustmentMethod")?.value ===
+                                                      "QUANTITY_WISE"
+                                                        ? stockGroupClosingBalance?.closing?.closing?.quantity ?? 0
+                                                        : giddhRoundOff(
+                                                              stockGroupClosingBalance?.closing.closing?.amount ?? 0,
+                                                              this.giddhBalanceDecimalPlaces
+                                                          )
+                                                    : 0
+                                            }}
+                                        </td>
+
+                                        <td class="px-3" width="15.625%" class="text-right">
+                                            {{
+                                                stockGroupClosingBalance?.changeValue
+                                                    ? stockGroupClosingBalance?.changeValue
+                                                    : 0
+                                            }}
+                                        </td>
+                                        <td class="px-4" width="15.625%" class="text-right">
+                                            {{
+                                                stockGroupClosingBalance?.newValue
+                                                    ? stockGroupClosingBalance?.newValue
+                                                    : 0
+                                            }}
+                                        </td>
+                                    </tr>
+                                </table>
+                            </mat-expansion-panel-header>
+                            <table
+                                *ngIf="dataSource?.data?.length && !(adjustInventoryInProgress$ | async)"
+                                mat-table
+                                [dataSource]="dataSource"
+                                class="table basic giddh-table on-mobile-view w-100 table-border-none"
+                            >
+                                <ng-container matColumnDef="select">
+                                    <th mat-header-cell *matHeaderCellDef width="20%">
+                                        <mat-checkbox
+                                            color="primary"
+                                            *ngIf="isEntityStockGroup"
+                                            [checked]="true"
+                                            [disabled]="isEntityStockGroup"
+                                            [aria-label]="checkboxLabel()"
```

Comment:
use of aria-label?

Reply:
Need for accessibility for dynamic content  

public checkboxLabel(row?: any): string {
        if (!row) {
            return `${this.isAllSelected() ? 'deselect' : 'select'} 'all'`;
        }
        return `${this.selection.isSelected(row) ? 'deselect' : 'select'} row ${row + 1}`;
    }